dev/com.ibm.ws.security.csiv2.common/src/com/ibm/ws/transport/iiop/security/config/tss/TSSSASMechConfig.java

```diff
@@ -231,7 +231,13 @@ void toString(String spaces, StringBuilder buf) {
         buf.append(moreSpaces).append("REQUIRES: ").append(ConfigUtil.flags(requires)).append("\n");
         for (Iterator<TSSServiceConfigurationConfig> iterator = privilegeAuthorities.iterator(); iterator.hasNext();) {
             TSSServiceConfigurationConfig tssServiceConfigurationConfig = iterator.next();
-            tssServiceConfigurationConfig.toString(moreSpaces, buf);
+            if (tssServiceConfigurationConfig instanceof TSSGeneralNameConfig) {
```

Comment:
If possible, please change these lines (234-240) to,

tssServiceConfigurationConfig.toString();

It will call the right method in the subclasses without casting them and prevent future changes if a new subclass is introduced.

Reply:
done.